neqo-transport/src/connection/tests/stream.rs

```diff
@@ -582,6 +618,103 @@ fn simultaneous_stop_sending_and_reset() {
     );
 }
 
+/// Make a stream data or control frame arrive after the stream has been used and cleared.
+fn late_stream_related_frame(frame_type: u64) {
+    let mut client = default_client();
+    let mut server = default_server();
+    connect(&mut client, &mut server);
+
+    // Client creates two streams and sends some data on the second.
+    _ = client.stream_create(StreamType::BiDi).unwrap();
+    let stream_id = client.stream_create(StreamType::BiDi).unwrap();
```

Comment:
Why two streams?

Reply:
Leftover from when I was debugging whether the`index` addition did the right thing. Can remove now.